app/src/test/java/org/simple/clinic/bloodsugar/history/adapter/BloodSugarHistoryListItemTest.kt

```diff
@@ -0,0 +1,116 @@
+package org.simple.clinic.bloodsugar.history.adapter
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.simple.clinic.bloodsugar.BloodSugarReading
+import org.simple.clinic.bloodsugar.Fasting
+import org.simple.clinic.bloodsugar.PostPrandial
+import org.simple.clinic.bloodsugar.Random
+import org.simple.clinic.bloodsugar.history.adapter.BloodSugarHistoryListItem.BloodSugarHistoryItem
+import org.simple.clinic.bloodsugar.history.adapter.BloodSugarHistoryListItem.NewBloodSugarButton
+import org.simple.clinic.patient.PatientMocker
+import org.simple.clinic.util.TestUserClock
+import org.threeten.bp.Instant
+import org.threeten.bp.format.DateTimeFormatter
+import java.util.Locale
+import java.util.UUID
+
+class BloodSugarHistoryListItemTest {
+  private val userClock = TestUserClock()
+  private val locale = Locale.ENGLISH
+  private val dateFormatter = DateTimeFormatter.ofPattern("d-MMM-yyyy", locale)
+  private val timeFormatter = DateTimeFormatter.ofPattern("h:mm a", locale)
+
+  @Test
+  fun `if there is only one list item, then divider should be shown`() {
```

Comment:
It made sense to test the generation of dividers earlier because we were showing and hiding them manually. Given that we now will use the item decoration instead, does it still make to test all of these?